src/lib/components/shared/markdown.svelte

```diff
@@ -73,3 +69,9 @@
         {@html html}
     {/await}
 </markdown>
+
+<style lang="postcss">
+    :global(pre) {
+        @apply flex bg-surface-400/50 md:rounded-[0.5vw] md:p-3 md:text-[0.9vw];
+    }
+</style>
```

Comment:
you forgot to add `unstyled` class?

![Screenshot from 2023-08-10 20-44-04](https://github.com/tokitou-san/CoreProject-V3-UI/assets/114811070/52f876a2-1226-4e37-8a0d-260fb1fd2bf4)

Reply:
Unstyled is not  a property

Reply:
Okay? add it in renderer?

Reply:
We need to modify everything by hand

Reply:
> Okay? add it in renderer?

can't. Cause we cant use escape

Reply:
Okay? sounds like a big plan lol, anyways this isnt working
fix it?

Reply:
Change Background color?